test/export/ValueSetExporter.test.ts

```diff
@@ -0,0 +1,229 @@
+    expect(exported.length).toBe(1);
+    expect(exported[0].name).toBe('BreakfastVS');
+    expect(exported[0].id).toBe('BreakfastVS');
+    expect(exported[0].title).toBe('Breakfast Values');
+    expect(exported[0].description).toBe('A value set for breakfast items');
+    expect(exported[0].url).toBe('http://example.com/ValueSet/BreakfastVS');
+  });
+
+  it('should export each value set once, even if export is called more than once', () => {
+    const breakfast = new FshValueSet('BreakfastVS');
+    doc.valueSets.set(breakfast.name, breakfast);
+    let exported = exporter.export();
+    expect(exported.length).toBe(1);
+    exported = exporter.export();
+    expect(exported.length).toBe(1);
+  });
+
+  it('should export a value set that includes a component from a system', () => {
+    const valueSet = new FshValueSet('DinnerVS');
+    const component = new ValueSetConceptComponent(true);
+    component.from = { system: 'FOOD' };
+    valueSet.components.push(component);
+    doc.valueSets.set(valueSet.name, valueSet);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0].compose.include.length).toBe(1);
+    expect(exported[0].compose.include[0].system).toBe('FOOD');
+  });
+
+  it('should export a value set that includes a component from a value set', () => {
+    const valueSet = new FshValueSet('DinnerVS');
+    const component = new ValueSetConceptComponent(true);
+    component.from = { valueSets: ['HotFoodVS', 'ColdFoodVS'] };
+    valueSet.components.push(component);
+    doc.valueSets.set(valueSet.name, valueSet);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0].compose.include.length).toBe(1);
+    expect(exported[0].compose.include[0].valueSet).toEqual(['HotFoodVS', 'ColdFoodVS']);
+  });
+
+  it('should export a value set that includes a concept component with at least one concept', () => {
+    const valueSet = new FshValueSet('DinnerVS');
+    const component = new ValueSetConceptComponent(true);
+    component.from = { system: 'FOOD' };
+    component.concepts.push(new FshCode('Pizza', 'FOOD', 'Delicious pizza to share.'));
+    component.concepts.push(new FshCode('Salad', 'FOOD', 'Plenty of fresh vegetables.'));
+    valueSet.components.push(component);
+    doc.valueSets.set(valueSet.name, valueSet);
+    const exported = exporter.export();
+    expect(exported.length).toBe(1);
+    expect(exported[0].compose.include.length).toBe(1);
+    expect(exported[0].compose.include[0].system).toBe('FOOD');
+    expect(exported[0].compose.include[0].concept.length).toBe(2);
+    expect(exported[0].compose.include[0].concept[0].code).toBe('Pizza');
+    expect(exported[0].compose.include[0].concept[0].display).toBe('Delicious pizza to share.');
+    expect(exported[0].compose.include[0].concept[1].code).toBe('Salad');
+    expect(exported[0].compose.include[0].concept[1].display).toBe('Plenty of fresh vegetables.');
```

Comment:
If it's simpler for you, it's totally valid to do an object-based equality assertion.  I.e., instead of above, do:
```ts
expect(exported[0]).toEqual({
  compose: {
    include: [{
      system: 'FOOD',
      concept: [
        { code: 'Pizza', display: 'Delicious pizza to share.' },
        { code: 'Salad', display: 'Plenty of fresh vegetables.' }
      ]
    }]
});
```

This actually has the added benefit in that if the result contains any unexpected keys (like `"foo": "bar"`), the test will fail.  That said, I'm not saying you have to do this -- just saying you _can_ do it, and it might be simpler than a bunch of long expect statements.  Up to you.